.eslintrc.js

```diff
@@ -6,15 +6,25 @@
 /* eslint-env node */
 
 module.exports = {
-    extends: "ckeditor5",
-    env: {
-        "browser": true,
-        "webextensions": true
-    },
-    rules: {
-        'max-len': 'off'
-    },
-    parserOptions: {
-        ecmaVersion: 2018,
+  extends: [
+    'ckeditor5',
+    'plugin:react/recommended'
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true
     }
+  },
+  env: {
+    mocha: false
+  },
+  settings: {
+    react: {
+      version: '16.3'
```

Comment:
Can we update the React version there to `16.8.6` (newest)?

Reply:
We use Preact, not React. So we actually can't use React's newest features.